stable/opsportal/values.yaml

```diff
@@ -0,0 +1,53 @@
+bearerproxy:
+  image:
+    repository: "quay.io/joejulian/bearerproxy"
+    tag: "latest"
+    pullPolicy: "Always"
+
+opsportal:
+  ingress:
+    paths:
+      # kubernetes-dashboard
+      - backend:
+          serviceName: bearerproxy
```

Comment:
And this would need logic, too, to either select the bearerproxy if it's enabled, or opsportal-landing if it's not.

Reply:
Sure thing.